src/main/dialogs/quickSetStrip/qssDialog.js

```diff
@@ -63,6 +65,13 @@ fluid.defaults("gpii.app.qss", {
             args: ["{change}.value"],
             // it is shown at first
             excludeSource: "init"
+        },
+        isShown: {
+            funcName: "gpii.app.qss.appBarUpdate",
+            args: ["{that}", "{appBar}", "{that}.dialog", "{change}.value", "{that}.model.appBarQss"],
+            namespace: "appBar",
+            priority: "before:impl"
+            //excludeSource: "init"
```

Comment:
Can remove commented line or explain why it might come back

Reply:
removed